### ISSUE TITLE
[IDEA]💲Minimum spend to withdraw SFL 

### ISSUE THREAD

**Issue by SrCouve** [sunflower-land/sunflower-land#775]
This is a proposal that may not be very popular, but I believe that those who support the project will at least be interested in reading it, I understand that the goal of the game is to be an accessible game and I support this idea, but I think it needs to have a bigger barrier to avoid players who only intend to dump the token, when the accessibility door is opened the number of these players also increases considerably, and having people who only want to farm to dump is not a good thing for the project

I believe that another barrier to withdrawal besides fees should be implemented:

Option 1: Minimum amount spent in SFL to start withdrawing

Option 2 : A score for Decorative NFTS, where your amount of SFL loot is determined by the amount of score you have, the more Decoration NFTS you have the more you can withdraw, and each such Decoration NFT has a score different depending on its cost (IN SFL), I think it would be a good way to give them the function, to stop people who just want to cash out and don't believe in the project, and a big headache for the multis

I understand that we have rates and Halvings, but outside our community I see a lot of influencers and players talking about farming just to dump and I believe we need another barrier in relation to this already implemented on the 9th, pors it would be a way to make it difficult the life of the mults, the life of those who just want to dump, would improve the health of the game's economy, and would greatly value the player who loves the project and is investing in the future of the game

**Comment by LucasNunesMendes** [comment]
> This is a proposal that may not be very popular, but I believe that those who support the project will at least be interested in reading it, I understand that the goal of the game is to be an accessible game and I support this idea, but I think it needs to have a bigger barrier to avoid players who only intend to dump the token, when the accessibility door is opened the number of these players also increases considerably, and having people who only want to farm to dump is not a good thing for the project
> 
> I believe that another barrier to withdrawal besides fees should be implemented:
> 
> Option 1: Minimum amount spent in SFL to start withdrawing
> 
> Option 2 : A score for Decorative NFTS, where your amount of SFL loot is determined by the amount of score you have, the more Decoration NFTS you have the more you can withdraw, and each such Decoration NFT has a score different depending on its cost (IN SFL), I think it would be a good way to give them the function, to stop people who just want to cash out and don't believe in the project, and a big headache for the multis
> 
> I understand that we have rates and Halvings, but outside our community I see a lot of influencers and players talking about farming just to dump and I believe we need another barrier in relation to this already implemented on the 9th, pors it would be a way to make it difficult the life of the mults, the life of those who just want to dump, would improve the health of the game's economy, and would greatly value the player who loves the project and is investing in the future of the game


Bro, great idea! I'll add another point here.

The fees may vary also according to the player's farm verification.

Unverified farms pay 10% - 15% more to withdraw funds

**Comment by brian-man** [comment]
I'm seeing this assumption in both discord and github:
Why are people assuming/expecting that _decorative_ NFTs should somehow be required for all players to own?  That makes them, by definition, non-decorative and actually *required* which makes them even more valuable than utility NFTs.  Going along this train of thought, maybe just list the required things every player has to do and the prescribed order?

TLDR: You play your way, I'll play mine.  Please don't make "decorate" things "required".

**Comment by SuperSynapse** [comment]
@SrCouve So I've put a lot of time into thinking about this issue. In my opinion there is really only one path forward for the game's economy.

1) Allow people to willingly withdraw as much SFL/resources to sell as they want. (this is crypto after all)
 - They get immediate profit
 - Their yield goes down over time with halvenings
 - They are only able to farm low level crops
 - Continually watched for botting/multi-accounting


2) SFL Team continually puts out more Utility NFTs that stack, that allow the long-term investors to 'keep up' as halvenings happen.
 - Those who invested in NFTs will see their value continue up with a multiplier effect.
 
 For example, assume 1 NFT provides a 2x yield, a second NFT provides a 2x yield (400% boost total). 
 
If another 2x yield NFT comes out, all three NFTs are no longer worth a 2x yield value, they are worth a 4x yield value, as they would move the TOTAL yield from 400% to 800% (a 400% increase). 

If another 2x yield NFT came out, now each one is worth an 800% yield (as getting a fourth 2x NFT would give you a 1600% total increase in yield). This continues on as long as the game is alive, and those wanting to catch up are able to buy on the open market at the current value of NFTs relative to their SFL production value.
 
 - Those don't work toward NFTs quickly get left behind as halvenings continue
 - Those who don't work toward NFTs quickly loose out on new crops, and more profitable farm upgradable mechanics.
 - In the short term, when SFL dumps due to selling pressure, those who want to further invest in SFL have the opportunity to do so.

**Comment by SuperSynapse** [comment]
I'll put this in a separate post. I don't believe the issue with Sunflower-Land has anything to do with the selling pressure per-se, but the limit to 1 account when there are still significant amounts of players multi-accounting right now.

By following the rules, you are missing out.

**Comment by SuperSynapse** [comment]
Also, I'll be releasing a proposal soon that has another take on your Option 2 :D

**Comment by SrCouve** [comment]
> Estou vendo essa suposição tanto na discórdia quanto no github: Por que as pessoas estão assumindo/esperando que nFTs _decorativos_ devem de alguma forma ser necessários para todos os jogadores possuí-los? Isso os torna, por definição, não decorativos e realmente _necessários_, o que os torna ainda mais valiosos do que nFTs utilitários.
> 
> TLDR: Você joga do seu jeito; Vou jogar o meu. Por favor, não faça coisas "decorativas" "necessárias".

Your analysis makes no sense, has no conflict with the functional NFTs, because in this proposal the decorative ones are linked to the withdrwa, for you to Withdrwal you need to generate and in that the Function NFTS help, it also makes no sense for you to say that when a decorative NFT receives a function it is no longer decorative, we have numerous examples of games where decoration adds buffs and boosts status in the game

**Comment by SrCouve** [comment]
> Estou vendo essa suposição tanto na discórdia quanto no github: Por que as pessoas estão assumindo/esperando que nFTs _decorativos_ devem de alguma forma ser necessários para todos os jogadores possuí-los? Isso os torna, por definição, não decorativos e realmente _necessários_, o que os torna ainda mais valiosos do que nFTs utilitários.
> 
> TLDR: Você joga do seu jeito; Vou jogar o meu. Por favor, não faça coisas "decorativas" "necessárias".

Your analysis makes no sense, has no conflict with the functional NFTs, because in this proposal the decorative ones are linked to the withdrawal , for you to Withdrawal you need to generate and in that the Function NFTS help, it also makes no sense for you to say that when a decorative NFT receives a function it is no longer decorative, we have numerous examples of games where decoration adds buffs and boosts status in the game

**Comment by SrCouve** [comment]
> I'm seeing this assumption in both discord and github: Why are people assuming/expecting that _decorative_ NFTs should somehow be required for all players to own? That makes them, by definition, non-decorative and actually _required_ which makes them even more valuable than utility NFTs. Going along this train of thought, maybe just list the required things every player has to do and the prescribed order?
> 
> TLDR: You play your way; I'll play mine. Please don't make "decorative" things "required".

Your analysis makes no sense, has no conflict with the functional NFTs, because in this proposal the decorative ones are linked to the withdrawal, for you  Withdraw you need to generate and in that the Function NFTS help, it also makes no sense for you to say that when a decorative NFT receives a function it is no longer decorative, we have numerous examples of games where decoration adds buffs and boosts status in the game

**Comment by SrCouve** [comment]
the objective here is not only to give a function to the decorative NFTS, but also to make life difficult for the Multis, and to reduce the eviction

**Comment by brian-man** [comment]
> the objective here is not only to give a function to the decorative NFTS, but also to make life difficult for the Multis, and to reduce the eviction

Decorative NFTS already have a function: decoration.
